### PR TITLE
✨Implement Whopa InFeed Content-Recommendation Platform

### PR DESCRIPTION
**Pull request by fedemramato**
This pull request implement new Content-Recommendation Platform for new product: "Whopa InFeed"
When included with the amp-embed tag, it will display a list of recommended content related to the site.

### CODE REVIEW

**Comment by fedemramato** [comment]
Please @zhouyx check this PR

**Comment by zhouyx** [comment]
to @ampproject/wg-ads for review

**Comment by fedemramato** [comment]
@calebcordry please approve this PR

File: ads/whopainfeed.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/whopainfeed.js]: nit: 2020

File: ads/_config.js [hunk 2]
```diff
@@ -1106,6 +1106,11 @@ const adConfig = jsonConfiguration({
     ],
   },
 
+  'whopainfeed': {
+    prefetch: 'https://widget.infeed.com.ar/widget/widget-amp.js',
+    renderStartImplemented: true,
```
Comment (calebcordry) [inline ads/_config.js]: I see you have this flag marked as true, but when I run your example ad I do not see this method being called. Can you confirm you are calling this API in your 3p code?

**Comment by fedemramato** [comment]
@calebcordry help me please. Some checks fail, but in local tests was successfully. I merge master with my branch to try resolve, but it has not worked.

**Comment by calebcordry** [comment]
Thank you for the patience, and the contribution!

**Pull request merged**